pandas/tests/io/pytables/test_compat.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+
+import pandas as pd
+
+from pandas.util.testing import assert_frame_equal
+
+
+tables = pytest.importorskip('tables')
+
+
+@pytest.fixture
+def pytables_hdf5_file(tmp_path):
+    """Use PyTables to create a simple HDF5 file.
+
+    There is no need for temporary file cleanup: pytest's `tmp_path` fixture
```

Comment:
remove this comment as well. see my comment below. we don't want to introduce a different idiom for testing (but as I said you can open an issue for discussion).

Reply:
> you can open an issue for discussion

 https://github.com/pandas-dev/pandas/issues/26984 :)